index.bs

```diff
@@ -135,7 +135,24 @@ Issue: Monkeypatch `window.open()`.
 
 # Fetch monkeypatches # {#fetch-monkeypatches}
 
-Issue: Specify monkeypatches for source/trigger registration.
+<pre class="idl">
+dictionary AttributionReportingRequestOptions {
+  boolean eventSourceEligible = false;
+  boolean triggerEligible = false;
+};
+
+partial dictionary RequestInit {
+  AttributionReportingRequestOptions? attributionReporting;
+};
+
+partial interface XMLHttpRequest {
+  attribute AttributionReportingRequestOptions? attributionReporting;
```

Comment:
Apparently dictionaries cannot be used as attributes, so this won't work.

https://webidl.spec.whatwg.org/#idl-attributes